### PR TITLE
feat: First version of the layout covering all views

### PR DESCRIPTION
**Pull request by gustavop-dev**
## Description

This PR completes the overall layout for the project, ensuring that all main components are fully structured and styled using Vue.js and Tailwind CSS.

### What's included:
- Finalized the main layout structure for all pages and components.
- Implemented base styling across the entire project using Tailwind CSS, ensuring responsiveness and consistency.

### Next steps (for future PRs):
- **Test data:** Future pull requests should include test data in the frontend to simulate real scenarios, which will allow us to manage events and add new features more effectively.

### CODE REVIEW

File: frontend/src/components/layouts/SlideBar.vue [hunk 1]
```diff
@@ -0,0 +1,204 @@
+<template>
+    <TransitionRoot as="template" :show="sidebarOpen">
+        <Dialog class="relative z-50 lg:hidden" @close="sidebarOpen = false">
+          <TransitionChild as="template" enter="transition-opacity ease-linear duration-300" enter-from="opacity-0" enter-to="opacity-100" leave="transition-opacity ease-linear duration-300" leave-from="opacity-100" leave-to="opacity-0">
```
Comment (carlos18bp) [inline frontend/src/components/layouts/SlideBar.vue:4]: Line too large

File: frontend/src/components/layouts/SlideBar.vue [hunk 2]
```diff
@@ -0,0 +1,204 @@
...
+            <div class="fixed inset-0 bg-gray-900/80" />
+          </TransitionChild>
+  
+          <div class="fixed inset-0 flex">
+            <TransitionChild as="template" enter="transition ease-in-out duration-300 transform" enter-from="-translate-x-full" enter-to="translate-x-0" leave="transition ease-in-out duration-300 transform" leave-from="translate-x-0" leave-to="-translate-x-full">
```
Comment (carlos18bp) [inline frontend/src/components/layouts/SlideBar.vue:9]: Line too large

File: frontend/src/components/layouts/SlideBar.vue [hunk 3]
```diff
@@ -0,0 +1,204 @@
...
+              <DialogPanel class="relative mr-16 flex w-full max-w-xs flex-1">
+                <TransitionChild as="template" enter="ease-in-out duration-300" enter-from="opacity-0" enter-to="opacity-100" leave="ease-in-out duration-300" leave-from="opacity-100" leave-to="opacity-0">
```
Comment (carlos18bp) [inline frontend/src/components/layouts/SlideBar.vue:11]: Line too large

File: frontend/src/components/layouts/SlideBar.vue [hunk 4]
```diff
@@ -0,0 +1,204 @@
...
+                    <img class="h-16 w-auto" src="@/assets/images/logo/logo1.png" alt="G&M Consultores Juridicos" />
+                  </div>
+                  <nav class="flex flex-1 flex-col">
+                    <ul role="list" class="flex flex-1 flex-col gap-y-5">
+                      <li>
+                        <!-- Profile dropdown -->
+                        <Menu as="div" class="relative">
+                            <MenuButton class="-m-1.5 flex items-center p-1.5">
+                            <span class="sr-only">Open user menu</span>
+                            <img class="h-8 w-8 rounded-full bg-gray-50" src="https://images.unsplash.com/photo-1472099645785-5658abf4ff4e?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=facearea&facepad=2&w=256&h=256&q=80" alt="" />
```
Comment (carlos18bp) [inline frontend/src/components/layouts/SlideBar.vue:31]: Line too large

File: frontend/src/components/layouts/SlideBar.vue [hunk 5]
```diff
@@ -0,0 +1,204 @@
...
+                            <span class="flex items-center">
+                                <span class="ml-4 text-sm font-semibold leading-6 text-gray-900" aria-hidden="true">Tom Cook</span>
+                                <ChevronDownIcon class="ml-2 h-5 w-5 text-gray-400" aria-hidden="true" />
+                            </span>
+                            </MenuButton>
+                            <transition enter-active-class="transition ease-out duration-100" enter-from-class="transform opacity-0 scale-95" enter-to-class="transform opacity-100 scale-100" leave-active-class="transition ease-in duration-75" leave-from-class="transform opacity-100 scale-100" leave-to-class="transform opacity-0 scale-95">
```
Comment (carlos18bp) [inline frontend/src/components/layouts/SlideBar.vue:37]: Line too large

File: frontend/src/components/layouts/SlideBar.vue [hunk 6]
```diff
@@ -0,0 +1,204 @@
...
+                                <a :href="item.href" :class="[active ? 'bg-gray-50' : '', 'block px-3 py-1 text-sm leading-6 text-gray-900']">{{ item.name }}</a>
+                                </MenuItem>
+                            </MenuItems>
+                            </transition>
+                        </Menu>
+                      </li>
+                      <li>
+                        <ul role="list" class="-mx-2 space-y-1">
+                          <li v-for="item in navigation" :key="item.name">
+                            <a :href="item.href" :class="[item.current ? 'bg-gray-50 text-secondary' : 'text-gray-700 hover:bg-gray-50 hover:text-secondary', 'group flex gap-x-3 rounded-md p-2 text-sm font-semibold leading-6']">
```
Comment (carlos18bp) [inline frontend/src/components/layouts/SlideBar.vue:49]: Line too large

File: frontend/src/components/layouts/SlideBar.vue [hunk 7]
```diff
@@ -0,0 +1,204 @@
...
+                              <component :is="item.icon" :class="[item.current ? 'text-secondary' : 'text-gray-400 group-hover:text-secondary', 'h-6 w-6 shrink-0']" aria-hidden="true" />
```
Comment (carlos18bp) [inline frontend/src/components/layouts/SlideBar.vue:50]: Line too large

File: frontend/src/components/layouts/SlideBar.vue [hunk 8]
```diff
@@ -0,0 +1,204 @@
...
+                              {{ item.name }}
+                            </a>
+                          </li>
+                        </ul>
+                      </li>
+                      <li class="mt-auto">
+                        <a href="#" class="group -mx-2 flex gap-x-3 rounded-md p-2 text-sm font-semibold leading-6 text-gray-700 hover:bg-gray-50 hover:text-secondary">
+                          <QuestionMarkCircleIcon class="h-6 w-6 shrink-0 text-gray-400 group-hover:text-secondary" aria-hidden="true" />
```
Comment (carlos18bp) [inline frontend/src/components/layouts/SlideBar.vue:58]: Line too large

File: frontend/src/components/layouts/SlideBar.vue [hunk 9]
```diff
@@ -0,0 +1,204 @@
...
+            <img class="h-22 w-auto" src="@/assets/images/logo/logo1.png" alt="G&M Consultores Juridicos" />
+          </div>
+          <nav class="flex flex-1 flex-col">
+            <ul role="list" class="flex flex-1 flex-col gap-y-5">
+              <li>
+                <!-- Profile dropdown -->
+                <Menu as="div" class="relative">
+                    <MenuButton class="-m-1.5 flex items-center p-1.5">
+                    <span class="sr-only">Open user menu</span>
+                    <img class="h-8 w-8 rounded-full bg-gray-50" src="https://images.unsplash.com/photo-1472099645785-5658abf4ff4e?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=facearea&facepad=2&w=256&h=256&q=80" alt="" />
```
Comment (carlos18bp) [inline frontend/src/components/layouts/SlideBar.vue:85]: Line too large

File: frontend/src/components/layouts/SlideBar.vue [hunk 10]
```diff
@@ -0,0 +1,204 @@
...
+                    <span class="hidden lg:flex lg:items-center">
+                        <span class="ml-4 text-sm font-semibold leading-6 text-gray-900" aria-hidden="true">Tom Cook</span>
+                        <ChevronDownIcon class="ml-2 h-5 w-5 text-gray-400" aria-hidden="true" />
+                    </span>
+                    </MenuButton>
+                    <transition enter-active-class="transition ease-out duration-100" enter-from-class="transform opacity-0 scale-95" enter-to-class="transform opacity-100 scale-100" leave-active-class="transition ease-in duration-75" leave-from-class="transform opacity-100 scale-100" leave-to-class="transform opacity-0 scale-95">
```
Comment (carlos18bp) [inline frontend/src/components/layouts/SlideBar.vue:91]: Line too large

File: frontend/src/components/layouts/SlideBar.vue [hunk 11]
```diff
@@ -0,0 +1,204 @@
...
+                        <a :href="item.href" :class="[active ? 'bg-gray-50' : '', 'block px-3 py-1 text-sm leading-6 text-gray-900']">{{ item.name }}</a>
+                        </MenuItem>
+                    </MenuItems>
+                    </transition>
+                </Menu>
+              </li>
+              <li>
+                <ul role="list" class="-mx-2 space-y-1">
+                  <li v-for="item in navigation" :key="item.name">
+                    <a :href="item.href" :class="[item.current ? 'bg-selected-background text-secondary' : 'text-primary hover:bg-gray-50 hover:text-secondary', 'group flex gap-x-3 rounded-md p-2 text-sm font-semibold leading-6']">
```
Comment (carlos18bp) [inline frontend/src/components/layouts/SlideBar.vue:103]: Line too large

File: frontend/src/components/layouts/SlideBar.vue [hunk 12]
```diff
@@ -0,0 +1,204 @@
...
+                      <component :is="item.icon" :class="[item.current ? 'text-secondary' : 'text-primary group-hover:text-secondary', 'h-6 w-6 shrink-0']" aria-hidden="true" />
```
Comment (carlos18bp) [inline frontend/src/components/layouts/SlideBar.vue:104]: Line too large

File: frontend/src/components/layouts/SlideBar.vue [hunk 13]
```diff
@@ -0,0 +1,204 @@
...
+          </button>
+  
+          <!-- Separator -->
+          <div class="h-6 w-px bg-gray-200 lg:hidden" aria-hidden="true" />
+  
+          <div class="flex flex-1 gap-x-4 self-stretch lg:gap-x-6">
+            <form class="relative flex flex-1" action="#" method="GET">
+              <label for="search-field" class="sr-only">Buscar</label>
+              <MagnifyingGlassIcon class="pointer-events-none absolute inset-y-0 left-0 h-full w-5 text-gray-400" aria-hidden="true" />
+              <input id="search-field" class="block h-full w-full border-0 py-0 pl-8 pr-0 text-gray-900 placeholder:text-gray-400 focus:ring-0 sm:text-sm" placeholder="Buscar" type="search" name="search" />
```
Comment (carlos18bp) [inline frontend/src/components/layouts/SlideBar.vue:135]: Line too large

File: frontend/src/views/directory/DirectoryList.vue [hunk 14]
```diff
@@ -1,6 +1,70 @@
...
+                            placeholder="Buscar usuarios" 
+                            type="search" 
+                        />
+                    </div>
+                </div>
+            </div>
+            <!-- Directory by cards -->
+            <div class="grid grid-cols-2">
+                <ul role="list" class="divide-y divide-gray-100">
+                    <li v-for="person in people" :key="person.email" class="relative flex justify-between gap-x-6 px-4 py-5 hover:bg-gray-50 sm:px-6 lg:px-8">
```
Comment (carlos18bp) [inline frontend/src/views/directory/DirectoryList.vue:29]: Line too large

File: frontend/src/views/process/ProcessForm.vue [hunk 15]
```diff
@@ -1,6 +1,225 @@
...
+                                class="text-red-500">*</span></label>
+                        <div class="mt-2">
+                            <input type="text" name="plaintiff" id="plaintiff"
+                                class="block w-full rounded-md border-0 py-1.5 text-primary shadow-sm ring-1 ring-inset ring-gray-300 placeholder:text-gray-400 focus:ring-2 focus:ring-inset focus:ring-secondary sm:text-sm sm:leading-6" />
+                        </div>
+                    </div>
+                    <!-- Defendant form -->
+                    <div>
+                        <label for="defendant" class="block text-base font-medium leading-6 text-primary">Dte./Accionado
+                            <span class="text-red-500">*</span></label>
```
Comment (carlos18bp) [inline frontend/src/views/process/ProcessForm.vue:25]: Fix identation here

File: frontend/src/views/directory/DirectoryList.vue [hunk 16]
```diff
@@ -1,6 +1,70 @@
...
+                        <img class="h-12 w-12 flex-none rounded-full bg-gray-50" :src="person.imageUrl" alt="" />
+                        <div class="min-w-0 flex-auto">
+                        <p class="text-sm font-semibold leading-6 text-gray-900">
+                            <a :href="person.href">
+                            <span class="absolute inset-x-0 -top-px bottom-0" />
+                            {{ person.name }}
+                            </a>
+                        </p>
+                        <p class="mt-1 flex text-xs leading-5 text-gray-500">
+                            <a :href="`mailto:${person.email}`" class="relative truncate hover:underline">{{ person.email }}</a>
```
Comment (carlos18bp) [inline frontend/src/views/directory/DirectoryList.vue:40]: Line too large

File: frontend/src/views/process/ProcessForm.vue [hunk 17]
```diff
@@ -1,6 +1,225 @@
...
+                                        </li>
+                                    </ComboboxOption>
+                                </ComboboxOptions>
+                            </div>
+                        </Combobox>
+                    </div>
+                    <!-- Subcase form -->
+                    <div>
+                        <label for="defendant" class="block text-base font-medium leading-6 text-primary">Subcaso
+                            <span class="text-red-500">*</span></label>
```
Comment (carlos18bp) [inline frontend/src/views/process/ProcessForm.vue:71]: Fix identation here

File: frontend/src/views/process/ProcessForm.vue [hunk 18]
```diff
@@ -1,6 +1,225 @@
...
+                                class="block w-full rounded-md border-0 py-1.5 text-primary shadow-sm ring-1 ring-inset ring-gray-300 placeholder:text-gray-400 focus:ring-2 focus:ring-inset focus:ring-secondary sm:text-sm sm:leading-6" />
+                        </div>
+                    </div>
+                </div>
+                <!-- Second row -->
+                <div class="grid grid-cols-3 gap-3">
+                    <!-- Ref form -->
+                    <div>
+                        <label for="plaintiff" class="block text-base font-medium leading-6 text-primary">Radicado <span
+                                class="text-red-500">*</span></label>
```
Comment (carlos18bp) [inline frontend/src/views/process/ProcessForm.vue:83]: Fix identation here

File: frontend/src/views/process/ProcessForm.vue [hunk 19]
```diff
@@ -1,6 +1,225 @@
...
+                        <div class="mt-2">
+                            <input type="text" name="plaintiff" id="plaintiff"
+                                class="block w-full rounded-md border-0 py-1.5 text-primary shadow-sm ring-1 ring-inset ring-gray-300 placeholder:text-gray-400 focus:ring-2 focus:ring-inset focus:ring-secondary sm:text-sm sm:leading-6" />
+                        </div>
+                    </div>
+                    <!-- Authority form -->
+                    <div>
+                        <label for="plaintiff" class="block text-base font-medium leading-6 text-primary">Autoridad
+                            <span class="text-red-500">*</span></label>
```
Comment (carlos18bp) [inline frontend/src/views/process/ProcessForm.vue:92]: Fix identation here

**Review by carlos18bp — CHANGES_REQUESTED** [review]

File: frontend/src/views/process/ProcessForm.vue (hunk 19 above)
Comment (gustavop-dev) [reply frontend/src/views/process/ProcessForm.vue]: Done!

File: frontend/src/views/process/ProcessForm.vue (hunk 18 above)
Comment (gustavop-dev) [reply frontend/src/views/process/ProcessForm.vue]: Done!

File: frontend/src/views/process/ProcessForm.vue (hunk 17 above)
Comment (gustavop-dev) [reply frontend/src/views/process/ProcessForm.vue]: Done!

File: frontend/src/views/directory/DirectoryList.vue (hunk 16 above)
Comment (gustavop-dev) [reply frontend/src/views/directory/DirectoryList.vue]: Done!

File: frontend/src/views/process/ProcessForm.vue (hunk 15 above)
Comment (gustavop-dev) [reply frontend/src/views/process/ProcessForm.vue]: Done!

File: frontend/src/views/directory/DirectoryList.vue (hunk 14 above)
Comment (gustavop-dev) [reply frontend/src/views/directory/DirectoryList.vue]: Done!

File: frontend/src/components/layouts/SlideBar.vue (hunk 13 above)
Comment (gustavop-dev) [reply frontend/src/components/layouts/SlideBar.vue]: Done!

File: frontend/src/components/layouts/SlideBar.vue (hunk 12 above)
Comment (gustavop-dev) [reply frontend/src/components/layouts/SlideBar.vue]: Done!

File: frontend/src/components/layouts/SlideBar.vue (hunk 11 above)
Comment (gustavop-dev) [reply frontend/src/components/layouts/SlideBar.vue]: Done!

File: frontend/src/components/layouts/SlideBar.vue (hunk 10 above)
Comment (gustavop-dev) [reply frontend/src/components/layouts/SlideBar.vue]: Done!

File: frontend/src/components/layouts/SlideBar.vue (hunk 9 above)
Comment (gustavop-dev) [reply frontend/src/components/layouts/SlideBar.vue]: Done!

File: frontend/src/components/layouts/SlideBar.vue (hunk 8 above)
Comment (gustavop-dev) [reply frontend/src/components/layouts/SlideBar.vue]: Done!

File: frontend/src/components/layouts/SlideBar.vue (hunk 7 above)
Comment (gustavop-dev) [reply frontend/src/components/layouts/SlideBar.vue]: Done!

File: frontend/src/components/layouts/SlideBar.vue (hunk 1 above)
Comment (gustavop-dev) [reply frontend/src/components/layouts/SlideBar.vue]: Done!

File: frontend/src/components/layouts/SlideBar.vue (hunk 2 above)
Comment (gustavop-dev) [reply frontend/src/components/layouts/SlideBar.vue]: Done!

File: frontend/src/components/layouts/SlideBar.vue (hunk 3 above)
Comment (gustavop-dev) [reply frontend/src/components/layouts/SlideBar.vue]: Done!

File: frontend/src/components/layouts/SlideBar.vue (hunk 4 above)
Comment (gustavop-dev) [reply frontend/src/components/layouts/SlideBar.vue]: Done!

File: frontend/src/components/layouts/SlideBar.vue (hunk 5 above)
Comment (gustavop-dev) [reply frontend/src/components/layouts/SlideBar.vue]: Done!

File: frontend/src/components/layouts/SlideBar.vue (hunk 6 above)
Comment (gustavop-dev) [reply frontend/src/components/layouts/SlideBar.vue]: Done!

File: frontend/src/views/process/ProcessDetail.vue [hunk 20]
```diff
@@ -1,6 +1,201 @@
...
+                <div class="flex items-center justify-between">
+                    <div class="hidden sm:flex sm:flex-1 sm:items-center sm:justify-between">
+                        <div>
+                            <nav class="isolate inline-flex -space-x-px rounded-md shadow-sm" aria-label="Pagination">
+                                <a href="#"
+                                    class="relative inline-flex items-center rounded-l-md px-2 py-2 text-gray-400 ring-1 ring-inset ring-gray-300 hover:bg-gray-50 focus:z-20 focus:outline-offset-0">
+                                    <span class="sr-only">Anterior</span>
+                                    <ChevronLeftIcon class="h-5 w-5" aria-hidden="true" />
+                                </a>
+                                <!-- Current: "z-10 bg-indigo-600 text-white focus-visible:outline focus-visible:outline-2 focus-visible:outline-offset-2 focus-visible:outline-indigo-600", Default: "text-gray-900 ring-1 ring-inset ring-gray-300 hover:bg-gray-50 focus:outline-offset-0" -->
```
Comment (carlos18bp) [inline frontend/src/views/process/ProcessDetail.vue:158]: Remove comment

File: frontend/src/components/layouts/SlideBar.vue [hunk 21]
```diff
@@ -0,0 +1,241 @@
...
+                  alt="" />
+                <span class="hidden lg:flex lg:items-center">
+                  <span class="ml-4 text-sm font-semibold leading-6 text-gray-900" aria-hidden="true">Tom Cook</span>
+                  <ChevronDownIcon class="ml-2 h-5 w-5 text-gray-400" aria-hidden="true" />
+                </span>
+              </MenuButton>
+              <transition enter-active-class="transition ease-out duration-100"
+                enter-from-class="transform opacity-0 scale-95" enter-to-class="transform opacity-100 scale-100"
+                leave-active-class="transition ease-in duration-75" leave-from-class="transform opacity-100 scale-100"
+                leave-to-class="transform opacity-0 scale-95">
```
Comment (carlos18bp) [inline frontend/src/components/layouts/SlideBar.vue:116]: Same comment as above, separate each attribute on a different line

File: frontend/src/components/layouts/SlideBar.vue [hunk 22]
```diff
@@ -0,0 +1,241 @@
...
+      </button>
+
+      <!-- Separator -->
+      <div class="h-6 w-px bg-gray-200 lg:hidden" aria-hidden="true" />
+
+      <div class="flex flex-1 gap-x-4 self-stretch lg:gap-x-6">
+        <form class="relative flex flex-1" action="#" method="GET">
+          <label for="search-field" class="sr-only">Buscar</label>
+          <MagnifyingGlassIcon class="pointer-events-none absolute inset-y-0 left-0 h-full w-5 text-gray-400"
+            aria-hidden="true" />
```
Comment (carlos18bp) [inline frontend/src/components/layouts/SlideBar.vue:169]: Better:
```
<MagnifyingGlassIcon 
    class="pointer-events-none absolute inset-y-0 left-0 h-full w-5 text-gray-400"
    aria-hidden="true" />
```

File: frontend/src/views/directory/DirectoryList.vue [hunk 23]
```diff
@@ -1,6 +1,66 @@
...
+                            <div class="min-w-0 flex-auto">
+                                <p class="text-sm font-semibold leading-6 text-gray-900">
+                                    <a :href="person.href">
+                                        <span class="absolute inset-x-0 -top-px bottom-0" />
+                                        {{ person.name }}
+                                    </a>
+                                </p>
+                                <p class="mt-1 flex text-xs leading-5 text-gray-500">
+                                    <a :href="`mailto:${person.email}`" class="relative truncate hover:underline">{{
+                                        person.email }}</a>
```
Comment (carlos18bp) [inline frontend/src/views/directory/DirectoryList.vue:36]: Better:
```
<a 
    :href="`mailto:${person.email}`" 
    class="relative truncate hover:underline">
    {{ person.email }}
</a>
```

File: frontend/src/views/process/ProcessForm.vue [hunk 24]
```diff
@@ -1,6 +1,242 @@
...
+                    <!-- Plaintiff form -->
+                    <div>
+                        <label for="plaintiff"
+                            class="block text-base font-medium leading-6 text-primary">
+                                Dte./Accionante 
+                                <span class="text-red-500">*</span></label>
+                        <div class="mt-2">
+                            <input type="text" name="plaintiff" id="plaintiff"
+                                class="block w-full rounded-md border-0 py-1.5 text-primary shadow-sm ring-1 ring-inset ring-gray-300 placeholder:text-gray-400 focus:ring-2 focus:ring-inset focus:ring-secondary sm:text-sm sm:leading-6"
+                            />
```
Comment (carlos18bp) [inline frontend/src/views/process/ProcessForm.vue:21]: Same comment as above, separate each attribute on a different line

File: frontend/src/views/process/ProcessForm.vue [hunk 25]
```diff
@@ -1,6 +1,242 @@
...
+                    <!-- Defendant form -->
+                    <div>
+                        <label for="defendant" class="block text-base font-medium leading-6 text-primary">
+                            Dte./Accionado
+                            <span class="text-red-500">*</span>
+                        </label>
+                        <div class="mt-2">
+                            <input type="text" name="defendant" id="defendant"
+                                class="block w-full rounded-md border-0 py-1.5 text-primary shadow-sm ring-1 ring-inset ring-gray-300 placeholder:text-gray-400 focus:ring-2 focus:ring-inset focus:ring-secondary sm:text-sm sm:leading-6"
+                            />
```
Comment (carlos18bp) [inline frontend/src/views/process/ProcessForm.vue:33]: Same comment as above, separate each attribute on a different line

File: frontend/src/views/process/ProcessForm.vue [hunk 26]
```diff
@@ -1,6 +1,242 @@
...
+                            <ComboboxLabel class="block text-base font-medium leading-6 text-primary">
+                                Tipo de caso
+                                <span class="text-red-500">*</span>
+                            </ComboboxLabel>
+                            <div class="relative mt-2">
+                                <ComboboxInput
+                                    class="w-full rounded-md border-0 bg-white py-1.5 pl-3 pr-10 text-primary shadow-sm ring-1 ring-inset ring-gray-300 focus:ring-2 focus:ring-inset focus:ring-secondary sm:text-sm sm:leading-6"
+                                    @change="query = $event.target.value" @blur="query = ''"
+                                    :display-value="(caseType) => caseType?.name" 
+                                />
```
Comment (carlos18bp) [inline frontend/src/views/process/ProcessForm.vue:48]: Same comment as above, separate each attribute on a different line
Down `@blur="query = ''"` statement

File: frontend/src/views/process/ProcessForm.vue [hunk 27]
```diff
@@ -1,6 +1,242 @@
...
+                    <!-- Subcase form -->
+                    <div>
+                        <label for="defendant" class="block text-base font-medium leading-6 text-primary">
+                            Subcaso
+                            <span class="text-red-500">*</span>
+                        </label>
+                        <div class="mt-2">
+                            <input type="text" name="defendant" id="defendant"
+                                class="block w-full rounded-md border-0 py-1.5 text-primary shadow-sm ring-1 ring-inset ring-gray-300 placeholder:text-gray-400 focus:ring-2 focus:ring-inset focus:ring-secondary sm:text-sm sm:leading-6" 
+                            />
```
Comment (carlos18bp) [inline frontend/src/views/process/ProcessForm.vue:84]: Same comment as above, separate each attribute on a different line

File: frontend/src/views/process/ProcessForm.vue [hunk 28]
```diff
@@ -1,6 +1,242 @@
...
+                    <!-- Ref form -->
+                    <div>
+                        <label for="plaintiff" class="block text-base font-medium leading-6 text-primary">
+                            Radicado 
+                            <span class="text-red-500">*</span>
+                        </label>
+                        <div class="mt-2">
+                            <input type="text" name="plaintiff" id="plaintiff"
+                                class="block w-full rounded-md border-0 py-1.5 text-primary shadow-sm ring-1 ring-inset ring-gray-300 placeholder:text-gray-400 focus:ring-2 focus:ring-inset focus:ring-secondary sm:text-sm sm:leading-6"
+                            />
```
Comment (carlos18bp) [inline frontend/src/views/process/ProcessForm.vue:99]: Same comment as above, separate each attribute on a different line

File: frontend/src/views/process/ProcessForm.vue [hunk 29]
```diff
@@ -1,6 +1,242 @@
...
+                    <!-- Authority form -->
+                    <div>
+                        <label for="plaintiff" class="block text-base font-medium leading-6 text-primary">
+                            Autoridad
+                            <span class="text-red-500">*</span>
+                        </label>
+                        <div class="mt-2">
+                            <input type="text" name="plaintiff" id="plaintiff"
+                                class="block w-full rounded-md border-0 py-1.5 text-primary shadow-sm ring-1 ring-inset ring-gray-300 placeholder:text-gray-400 focus:ring-2 focus:ring-inset focus:ring-secondary sm:text-sm sm:leading-6"
+                            />
```
Comment (carlos18bp) [inline frontend/src/views/process/ProcessForm.vue:111]: Same comment as above, separate each attribute on a different line

File: frontend/src/views/process/ProcessForm.vue [hunk 30]
```diff
@@ -1,6 +1,242 @@
...
+                            <ComboboxLabel class="block text-sm font-medium leading-6 text-primary">
+                                Usuario
+                                <span class="text-red-500">*</span>
+                            </ComboboxLabel>
+                            <div class="relative mt-2">
+                                <ComboboxInput
+                                    class="w-full rounded-md border-0 bg-white py-1.5 pl-3 pr-12 text-primary shadow-sm ring-1 ring-inset ring-gray-300 focus:ring-2 focus:ring-inset focus:ring-secondary sm:text-sm sm:leading-6"
+                                    @change="query = $event.target.value" @blur="query = ''"
+                                    :display-value="(person) => person?.name" />
+                                <ComboboxButton
```
Comment (carlos18bp) [inline frontend/src/views/process/ProcessForm.vue:126]: Same comment as above, separate each attribute on a different line
Down `@blur="query = ''"` statement

**Pull request merged**